.github/workflows/branch.yaml

```diff
@@ -0,0 +1,43 @@
+---
+
+#########################
+#########################
+## Deploy Docker Image  to ECR##
+#########################
+#########################
+
+name: CI Pipeline for lazy dotnet image push to ECR 
+on:
+  release:
```

Comment:
Workflow file says branch. Here it says release. Can you fix that?
Also, will there be a docker image created for every push. 
I believe it should be created so that we can test without creating a release everytime.

Reply:
This is addressed also created branch workflow.